llvm/test/MC/AMDGPU/misaligned-vgpr-tuples-err.s

```diff
@@ -23,74 +23,74 @@ global_load_dwordx4 a[1:4], v[0:1], off
 
 
 image_load v[1:2], v2, s[0:7] dmask:0x3 unorm
-// GFX90A: :[[@LINE-1]]:{{[0-9]+}}: error: invalid register class: vgpr tuples must be 64 bit aligned
```

Comment:
It seems like the previous error message is more clear?

Reply:
See commit message. The way these work is pretty much a hack and bypasses the automatic diagnostics we would get if we matched instructions properly